### PR TITLE
NMA-1066  Custodial Swaps | Buy Dash

### PR DESCRIPTION
**Pull request by hadia**
user who has a Coinbase account I would like to be able to swap my crypto holdings in my Coinbase account for Dash in my Dash Wallet.

## How Has This Been Tested?

- [ ] QA (Mobile Team)

## Checklist:

- [ ] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

File: common/src/main/res/values/styles.xml
Comment (Syn-McJ) [inline common/src/main/res/values/styles.xml:858]: I don't think this needed to be renamed. Dash Blue is our primary Blue color.

File: features/exploredash/src/androidTest/java/org/dash/wallet/features/exploredash/TestUtils.kt
Comment (Syn-McJ) [inline features/exploredash/src/androidTest/java/org/dash/wallet/features/exploredash/TestUtils.kt:13]: Why are these test classes in the Explore Dash?

File: integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/di/CoinBaseModule.kt
Comment (Syn-McJ) [inline integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/di/CoinBaseModule.kt:67]: `@Provides` is for providing an existing instance of a class. If a new instance is created, you can use `@Binds` - see `ExploreDashModule`.

File: integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/convert_currency/ConvertView.kt
Comment (Syn-McJ) [inline integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/convert_currency/ConvertView.kt:144]: This will ignore localization rules for the currency in front/after the amount. `fiatToString` method would be better suited here.

File: integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/CoinbaseConvertCryptoViewModel.kt
Comment (Syn-McJ) [inline integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/CoinbaseConvertCryptoViewModel.kt:86]: The failure state is ignored which seems weird.

**Review by Syn-McJ — APPROVED** [review]
Looks good to me, I have some comments that can probably be fixed later. I'll leave it to @HashEngineering

**Review by HashEngineering — CHANGES_REQUESTED** [review]
Yes, any comments can be fixed in the next PR.

There is a merge conflict which I can fix before merging later today.

**Review by HashEngineering — APPROVED** [review]

**Pull request merged**
